python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -122,8 +122,19 @@ export default function EditorContainer({
   const debouncedUpdatePrompt = useMemo(
     () =>
       debounce(
-        (promptName: string, newPrompt: Prompt) =>
-          updatePromptCallback(promptName, newPrompt),
+        async (
+          promptName: string,
+          newPrompt: Prompt,
+          resFn?: (aiconfigRes: AIConfig) => void
```

Comment:
nit: can we call this something a bit more clear? `res` to me I've never really liked because it can mean either `result` or `response`. `responseCallbackHandler` or `onCompletionHandler` are some examples I'd use

Reply:
Oh yea, this seemed to me not clear that it was `onSuccess` or `onCompletion`. I think if you intend this to be `onSuccess` only then that makes sense!